include/pluto_constants.h

```diff
@@ -1019,6 +1032,16 @@ enum pluto_exit_code {
 };
 
 extern enum_names pluto_exit_code_names;
+/*
+ * EXPIRE type events from the kernel.
+ * Based on these, different actions can be taken, eg skipping delete SPI
+ */
+enum sa_expire_kind {
+	/* 0 reserved */
+	SA_ACTIVE = 1,
+	SA_SOFT_EXPIRED = 2,
```

Comment:
this is probably nit.  I would argue after SA_SOFT_EXPIRED it is uncertain.
because the HARD expire message may be in the Q and pluto would run into unpredictable behavior.
also feel that is in nit picking territory. Lets leave it for now.